internal/db/postgres/cmd/dump.go

```diff
@@ -221,7 +221,7 @@ func (d *Dump) schemaOnlyDump(ctx context.Context, tx pgx.Tx) error {
 func (d *Dump) dataDump(ctx context.Context) error {
 	// TODO: You should use pointer to dumpers.DumpTask instead
 	tasks := make(chan dumpers.DumpTask, d.pgDumpOptions.Jobs)
-	result := make(chan dump.Entry, d.pgDumpOptions.Jobs)
+	result := make(chan dump_objects.Entry, d.pgDumpOptions.Jobs)
```

Comment:
Not quite critical, but underscores in func/var names are considered against Go style